build.sbt

```diff
@@ -395,22 +414,30 @@ lazy val plugin = project
   )
   .enablePlugins(ScriptedPlugin)
 
+lazy val jsApi =
+  project
+    .in(file("mdoc-js-api"))
```

Comment:
Can we name this `mdoc-js-interfaces` for consistency with mtags-interfaces? (and other artifacts using the `-interfaces` suffix)